### ISSUE TITLE
Specific prices can reduce product cost to zero with £ / % error

### ISSUE THREAD

**Issue by gingerling** [thirtybees/thirtybees#1084]
On specific prices section of prices tab on a product, if e.g. you try and do 20% discount on a product costing less than £20.00 but accidentally do £20 discount because you don't change the dropdown to percent, it allows customers to buy at £0.00 - there should really be an error for the admin saving the rule, in this case

**Comment by gingerling** [comment]
You are also allowed to do e.g. a £60 discount on a £3 product (I can see maybe someone would want to do a £20 discount on a £20 product for some reason, but this second scenario really should flag up an error)

**Comment by getdatakick** [comment]
I'm not sure this is a bug. I'm also not convinced it should be strictly forbidden to set such discounts. But I think it would be nice to have some warning  to inform user about the impact

**Comment by gingerling** [comment]
I agree, it shouldn't be forbidden, but a warning would be prudent as it's easy to make a mistake with the percent/pounds dropdown. I think that, technically speaking, if you made a mistake where you charged zero for a product you would have to honour it (in the UK). I'm not a lawyer, but I know that most people believe strongly that you must sell a product at the price advertised, even if the price is wrong. In my case the customer did not challenge it but I know many people who would have done - I have challenged this myself in shops before even!
